source/documentation/at-rules/import.html.md.erb

```diff
@@ -531,6 +531,56 @@ nice API for users of the new module system.
   @forward "reset" as reset-*
 <% end %>
 
+#### Configuring Modules Through Imports
+
+<% impl_status dart: '1.24.0', libsass: false, ruby: false %>
+
+You can configure modules that are loaded through an `@import` simply by
+defining global variables prior the `@import` that first loads that module.
+
+Note that changes to the configuration after the module is first loaded will be
+ignored.
```

Comment:
I find this kind of confusing. Either elaborate on it by describing exactly the situation in which this might be a problem (possibly within a `<% heads_up do %>` block), or leave it out if you don't think it's likely to be a substantial problem in practice.